docs/core/toc.yml

```diff
@@ -83,16 +84,9 @@
   displayName: app compatibility
   href: compatibility/
 - name: Tutorials
-  href: tutorials/index.md
   items:
-  - name: Templates for the CLI
-    items:
-    - name: 1 - Create an item template
-      href: tutorials/cli-templates-create-item-template.md
-    - name: 2 - Create a project template
-      href: tutorials/cli-templates-create-project-template.md
-    - name: 3 - Create a template pack
-      href: tutorials/cli-templates-create-template-pack.md
+  - name: Index of tutorials
```

Comment:
not super sold on this name, but also don't have good suggestions here.

Reply:
I changed it to follow the pattern elsewhere, i.e., the overview entry for the section is labeled "Overview".